x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/overview_tab.tsx

```diff
@@ -47,6 +58,16 @@ const getDetailsList = (data: CspFinding) => [
     }),
     description: data.rule.section,
   },
+  {
+    title: i18n.translate('xpack.csp.findings.findingsFlyout.overviewTab.indexTitle', {
+      defaultMessage: 'Index',
+    }),
+    description: discoverIndexLink ? (
+      <EuiLink href={discoverIndexLink}>{INDEX_LINK_NAME}</EuiLink>
+    ) : (
+      INDEX_LINK_NAME
+    ),
```

Comment:
opted for plain text rather than disabled. feels like disabled is more for buttons, i cant recall a disabled link ui. some links can be gray it doesn't mean they are disabled or won't work. using plain text removes the underline and its clear its not clickable